.gitignore

```diff
@@ -27,6 +27,7 @@
 
 # PyCharm files
 /.idea/
+*.iml
```

Comment:
I am working with IntelliJ, which needs and always produces this iml file for storing the module information. It may be less common than PyCharm but I don't see the benefit of installing PyCharm as well, given that the main difference between the two programs is the list of preinstalled plugins. I simply installed the python plugin to my IntelliJ and have a a PyCharm clone.

If that does not convince you we can leave it out though, I just wanted to explain how it comes that I added the line.